app/views/searches/results.csv.erb

```diff
@@ -1,4 +1,4 @@
-"Name","Account Number","Year Carried Over","Credit Details","Date Account Opened"
+"Name","Account Number","Year Carried Over","Credit Details","Date Account Opened","Case Link"
 <% @cases.each do |kase| %>
-<%= raw kase.case_name %>,<%= kase.account_number %>,<%= kase.year_carried_over %>,<%= raw kase.credit_details %>,<%= kase.date_account_opened %>
+<%= raw kase.case_name %>,<%= kase.account_number %>,<%= kase.year_carried_over %>,<%= raw kase.credit_details %>,<%= kase.date_account_opened %>,<%= "https://find-unclaimed-court-money.service.justice.gov.uk/#{(case_path(kase.account_number))}" %>
```

Comment:
If are using a URL helper like `case_path` but you need a full URL then you can use `_url` instead of `_path` (mentioned here - https://guides.rubyonrails.org/routing.html#path-and-url-helpers).

That will then use the host that the user is currently accessing it from and you don't need to hardcode it.

Reply:
do I need to create a test for this change?

Reply:
We have a test for the CSV generally but not for the particular details. It might be worth creating a view spec similar to spec/views/searches/results.html.erb_spec.rb for it.

Reply:
I've created the test for the csv file